apisix/utils/log-util.lua

```diff
@@ -119,13 +120,32 @@ local function get_full_log(ngx, conf)
     }
 
     if conf.include_req_body then
```

Comment:
I think we can change the current logic to
```
if include_req_body || eval(include_req_body_expr) {
  log req body
}
```
What is your opinion?

Reply:
In my opinion, i think `include_req_body` like a main switch, `include_req_body_expr` like a switch in condition. Close  `include_req_body`  will not log any request body data, and `include_req_body`'s semantic is compatible with  the previous version of kafka-logger, we are just only add a condition expr.